learningresource/models.py

```diff
@@ -18,11 +18,11 @@ class LearningResource(models.Model):
     title = models.CharField(max_length=255)
     url = models.TextField()
     description = models.TextField()
-    media_type_id = models.ForeignKey(MediaType, on_delete=models.CASCADE)
-    department_id = models.ForeignKey(Department, on_delete=models.CASCADE)
+    media_type = models.ForeignKey(MediaType, on_delete=models.CASCADE)
+    department = models.ForeignKey(Department, on_delete=models.CASCADE)
```

Comment:
Thanks for fixing it up. Looks good

Reply:
Okay, I also removed the 'ids' within the accounts tables within my branch.